### PR TITLE
Modified Files to fix find bugs issues.

### PR DESCRIPTION
**Pull request by jayavardhanvny**
Modified Files to fix find bugs issues. Let it be check by you, for further configuration of tomcat.

### CODE REVIEW

File: src/main/java/org/letuslearn/database/utility/SqlStatement.java [hunk 1]
```diff
@@ -54,6 +53,8 @@ public String getAllColumsString(Connection connection, String tableName) {
     } catch (SQLException e) {
       log.info("Check ur Columns parameter in Database metadata ");
```
Comment (prasadram) [inline src/main/java/org/letuslearn/database/utility/SqlStatement.java]: logging should be error

File: src/main/java/org/letuslearn/database/utility/TableMetadataUtility.java [hunk 2]
```diff
@@ -36,16 +36,21 @@ public int getColumnCount(String tableName) {
     try {
       Statement informationSchemaStatement = ConnectionProvider.getInstance().getConnection()
           .createStatement();
-
-      ResultSet resultCount = informationSchemaStatement.executeQuery(sqlSchemaQuery);
-      count = resultCount.getMetaData().getColumnCount();
-      log.info("Count is " + count);
+      try {
```
Comment (prasadram) [inline src/main/java/org/letuslearn/database/utility/TableMetadataUtility.java:39]: why these many nested trys

File: src/main/java/org/letuslearn/database/utility/SqlStatement.java [hunk 3]
```diff
@@ -32,16 +32,15 @@ public String getInsertStatment(String tableName) {
     String s = "";
```
Comment (prasadram) [inline src/main/java/org/letuslearn/database/utility/SqlStatement.java]: try to use abbreviated variable names

**Review by prasadram — CHANGES_REQUESTED** [review]
fix review comments

File: src/main/java/org/letuslearn/database/utility/TableMetadataUtility.java (hunk 2 above)
Comment (jayavardhanvny) [reply src/main/java/org/letuslearn/database/utility/TableMetadataUtility.java:39]: Able to close first result set then statement and connection.

**Pull request closed without merging**